css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  height: 60px;
+  width: 150px;
+}
+
+button:hover {
+  background-color: blue;
+  color: white;
+}
```

Comment:
Please make the colors the same as in the animation, and change the cursor's pointer as well:

---

![image](https://user-images.githubusercontent.com/16143771/151229504-76904dc2-3ffc-488d-a73a-23cbc6d12b82.png)

---

![image](https://user-images.githubusercontent.com/16143771/151229471-af8de231-184f-40d9-b72e-fa39d9121ff9.png)